classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    student.classroom = self
+    students << student
```

Comment:
- [ ] You created a variable `@students` on line 7 to be an empty array but on this highligted line 12 you are not referring to that variable, please kindly correct this line of code by useing `@students` :+1: :sparkles: